stl/inc/vector

```diff
@@ -2393,7 +2393,8 @@ public:
 
 #if _ITERATOR_DEBUG_LEVEL != 0
     _CONSTEXPR20 _Difference_type _Total_off(const _Mycont* _Cont) const noexcept {
-        return static_cast<_Difference_type>(_VBITS * (_Myptr - _Cont->_Myvec.data()) + _Myoff);
+        return static_cast<_Difference_type>(
+            static_cast<_Difference_type>(_VBITS) * (_Myptr - _Cont->_Myvec.data()) + _Myoff);

```

Comment:
There's `static_cast<_Difference_type>(_VBITS)` used three times, causing line wrap added in two of the occurrences. This might justify introducing a member constant of `_Difference_type` type.